modules/autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/AutofillJavascriptInterface.kt

```diff
@@ -79,6 +83,18 @@ class AutofillJavascriptInterface(
         }
     }
 
+    private fun filterRequestedSubtypes(request: AutofillDataRequest, credentials: List<LoginCredentials>): List<LoginCredentials> {
```

Comment:
So subtype corresponds as well to what the user clicked, so for this scenario:
- User opens trello
- User saves password without username
- User clicks on username field

Actual result: no prompt is shown

Expected result: There's a login saved for trello but without username, so shouldn't it show still?

When I click on password field, the autofill prompt appears though.

Reply:
I tested all the steps above and it works as expected. Not sure if the scenario I mentioned above is an edge case but at least we have a bottom sheet autofill design on how to display when the login saved has no username but has password  for the domain.

Reply:
Can we not these kinds of cases somewhere? Just to have the right expectation on unusual but not edge case-y scenarios.